minecraft-modpack-gtnh/minecraft-modpack-gtnh-docker.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "minecraft-modpack-gtnh-docker",
+  "display": "Minecraft Modpack - GT New Horizons (Docker)",
+  "type": "minecraft-java",
+  "install": [
+    {
+      "files": [
+        "http://downloads.gtnewhorizons.com/ServerPacks/GT_New_Horizons_${gtnh-ver}_SERVER_JAVA9.zip"
```

Comment:
Just a heads up, the server zip file names has changed

![image](https://user-images.githubusercontent.com/18713839/225198343-575b7f98-e087-4ff2-8e52-890623fa0ea0.png)


1 possible solution is to remove the version selection and instead have a textbox with instructions on where to get the zip url for the proper java/gtnh version you wish the server to be on.

Reply:
Updated with the new format.
Later today will try to change it to url or make an option for the /old/ versions, since its URL it's a bit different. Maybe just a "Is in the /old/ folder?", similar to [the forge pre-1.17 template](https://github.com/PufferPanel/templates/blob/a1fc5598e4b4e0688658920aba9b261174b71e56/minecraft-forge/minecraft-forge.json#L49)

I would prefer not to use a full URL since "rm *.zip" doesn't work (at least for me). And that would break the easy updating part of it.